vcnc-rest/static/Makefile.am

```diff
@@ -12,7 +12,7 @@
 all:
 	if [ ! -d $(top_srcdir)/static/swagger-ui ]; then \
 	  (cd $(top_srcdir)/static; \
-	  $(PROG_GIT) clone https://github.com/swagger-api/swagger-ui.git; \
+	  $(PROG_GIT) clone --depth 1 --branch v2.2.10 https://github.com/swagger-api/swagger-ui.git; \
```

Comment:
Third-party also has the disadvantage that it's only on Perforce.  It's useful for Go pipeline builds, but not so much for setting up developer environments, especially from GitHub.